ci/create-windows-image.ps1

```diff
@@ -0,0 +1,274 @@
+### Run from an Administrator command prompt with:
+### powershell -executionpolicy Unrestricted ci/create-image.sh
+### The machine will reboot after the script runs.
+
+### Required env vars:
+### $env:UNITY_SERIAL
+### $env:UNITY_USERNAME
+### $env:UNITY_PASSWORD
+
+# https://chocolatey.org/docs/how-to-setup-offline-installation
+$CHOCOLATEY_VERSION="0.10.11"
+$GIT_VERSION="2.18.0"
+$RESHARPER_CLI_VERSION="2018.1.2"
+$SPATIAL_VERSION="1.1.9"
+
+# From https://public-cdn.cloud.unity3d.com/hub/prod/releases-win32.json
+$UNITY_VERSION="2018.1.6f1"
+$UNITY_RELEASE_HASH="57cc34175ccf"
+
+# Direct link to installer at https://my.visualstudio.com/Downloads?q=Visual%20Studio%202017
+# VS Professional 15.7
+$VISUAL_STUDIO_HASH="09e0efec-0cfd-4015-b732-e8543589a6e0/1c89b3fa4ca0d287138916dca1625a0e"
+
+### To get a logfile in "c:\programdata\ssh\logs", add the following in the embedded script segment named $sshdConfigContent, below
+###     SyslogFacility LOCAL0
+###     LogLevel DEBUG
+
+### TODO: Install Improbable robots SSH keys for github
+### TODO: Install spatial service tokens
+
+Import-Module BitsTransfer
+$ErrorActionPreference = "Stop"
+
+mkdir c:\build
+
+Write-Host "Installing Chocolatey..."
+$env:chocolateyVersion = "$CHOCOLATEY_VERSION"
+Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+
+Write-Host "Installing git..."
+Start-Process -Wait -NoNewWindow -FilePath "choco" -ArgumentList "install", "git", "--yes", "--version=$GIT_VERSION"
+
+Write-Host "Installing Resharper CLI tools..."
+Start-Process -Wait -NoNewWindow -FilePath "choco" -ArgumentList "install", "resharper-clt.portable", "--yes", "--version=$RESHARPER_CLI_VERSION"
+
+Write-Host "Installing Spatial CLI..."
+Start-Process -Wait -NoNewWindow -FilePath "choco" -ArgumentList "install", "spatial", "--yes", "--version=$SPATIAL_VERSION"
+
+Start-BitsTransfer -Source "https://download.unity3d.com/download_unity/$UNITY_RELEASE_HASH/Windows64EditorInstaller/UnitySetup64-$UNITY_VERSION.exe" -Destination "c:\build\unity-installer.exe"
+Start-BitsTransfer -Source "https://download.unity3d.com/download_unity/$UNITY_RELEASE_HASH/TargetSupportInstaller/UnitySetup-Linux-Support-for-Editor-$UNITY_VERSION.exe" -Destination "c:\build\unity-linux-installer.exe"
+Start-BitsTransfer -Source "https://download.unity3d.com/download_unity/$UNITY_RELEASE_HASH/TargetSupportInstaller/UnitySetup-Mac-Mono-Support-for-Editor-$UNITY_VERSION.exe" -Destination "c:\build\unity-mac-installer.exe"
+Start-BitsTransfer -Source "https://download.visualstudio.microsoft.com/download/pr/$VISUAL_STUDIO_HASH/vs_professional.exe" -Destination "c:\build\vs-installer.exe"
+
+Write-Host "Installing Unity..."
+Start-Process -Wait -NoNewWindow -FilePath /build/unity-installer.exe -ArgumentList "/S"
```

Comment:
Since we're not specifying the target directory `/D`, this might break in the future if Unity every changes this.

Reply:
The odds of that are pretty low. I want us to do the default thing that Unity does to keep in line with what many users will do. And if Unity do, against all best practice and convention, change from `%PROGRAMFILES%\Unity`, then when we upgrade to that version, we update our scripts to reflect the new reality, rather than forcing the installation into where we want it to be.